torchcfm/conditional_flow_matching.py

```diff
@@ -432,9 +451,58 @@ def sample_location_and_conditional_flow(self, x0, x1, return_noise=False):
 
 
 class VariancePreservingConditionalFlowMatcher(ConditionalFlowMatcher):
+    """Albergo et al. 2023 stochastic interpolants class. This class inherits the
+    ConditionalFlowMatcher and override the compute_mu_t and
+    compute_conditional_flow functions in order to compute [3]'s stochastic interpolants.
```

Comment:
Can we call this the "trigonometric interplant" from [3]? I think [3] is more general than this (though this is the one they tested I believe.